package.json

```diff
@@ -4,11 +4,13 @@
   "main": "main.js",
   "types": "typings/n-express.d.ts",
   "scripts": {
-    "test": "make test",
+    "test": "dotcom-tool-kit test:local",
     "test:types": "tsc",
```

Comment:
**question (non-blocking):** would adding something like [this plugin](https://github.com/Financial-Times/dotcom-biz-ops-package-updater/tree/main/toolkit/typescript-check) to Tool Kit make sense? It does the job of `test:types` in n-express and would allow it to be done as part of `npm test` in future. I can see us needing it in a few places as we maintain a bunch of JavaScript libraries which we want to ensure are still type-safe.

Reply:
I just created a [PR](https://github.com/Financial-Times/dotcom-tool-kit/pull/338) to add a TypeScript plugin to Tool Kit that includes a task called [`TypeScriptTest`](https://github.com/Financial-Times/dotcom-tool-kit/blob/e79361c5b181d5901e91d2f59f05b31ff8edd5a0/plugins/typescript/src/tasks/typescript.ts#L42-L46) that should do what you describe

Reply:
Amazing! I'll check if this works for our case today